CONTRIBUTING.md

```diff
@@ -649,11 +649,11 @@ should be canceled.
 
 ### Maintainers
 
-- [Damien Mathieu](https://github.com/dmathieu), Elastic
+- [Damien Mathieu](https://github.com/dmathieu), Elastic ([GPG](https://keys.openpgp.org/search?q=408B8E44F0873832)
 - [David Ashpole](https://github.com/dashpole), Google
-- [Robert Pająk](https://github.com/pellared), Splunk
-- [Sam Xie](https://github.com/XSAM), Cisco/AppDynamics
-- [Tyler Yahn](https://github.com/MrAlias), Splunk
+- [Robert Pająk](https://github.com/pellared), Splunk ([GPG](https://keys.openpgp.org/search?q=0xE5F7C35A4DBE90C2))
+- [Sam Xie](https://github.com/XSAM), Splunk ([GPG](https://keys.openpgp.org/search?q=AEA033782371ABB18EE39188B8044925D6FEEBEA))
+- [Tyler Yahn](https://github.com/MrAlias), Splunk ([GPG](https://pgp.mit.edu/pks/lookup?search=0x46B0F3E1A8B1BA5A))
```

Comment:
`pgp.mit.edu` takes 80 seconds to return. Should we use `keys.openpgp.org`?

Reply:
I also found it to be unusable.

Reply:
I agree, see https://github.com/open-telemetry/opentelemetry-go/pull/6855#discussion_r2135346943